### ISSUE TITLE
The maximum network bandwidth is suboptimal.

### ISSUE THREAD

**Issue by IntegratedQuantum** [PixelGuys/Cubyz#419]
Raw TCP and raw UDP seem to have a bandwidth of 1 GB/s or more on localhost, when sending 60 MB worth of data.
If I replicate the same thing in Cubyz, by sending 60 MB at once, it takes around 4 seconds → 15 MB/s
Why is my custom protocol so slow compared to TCP? Is it the startup time?

**Comment by alexdesander** [comment]
Did you measure reliable or unreliable throughput?

That's a good way of determining where the problem could be.

**Comment by IntegratedQuantum** [comment]
Yeah, that's where my numbers come from. Though I didn't measure the unreliable throughput with my protocol, I think I did the test with another application and raw UDP. But I don't think that should be a problem, since I'm basically just using the raw socket API for unreliable UDP messages.

**Comment by alexdesander** [comment]
I mean this depends on whether you have a congestion control algorithm going, normally that would also throttle unreliable throughput.

Do you ack every packet individually or do you send sum acks?

Do you have some kind of sliding windoe going on?

**Comment by IntegratedQuantum** [comment]
I have congestion control, but it only throttles reliable packets.
I send bulk acks once every 100 ms.
I have a sliding window of up to 65536 packets of slightly less than 1500 bytes each, this should not be a problem here.
